build.rs

```diff
@@ -18,16 +18,16 @@ fn main() {
 fn add_resources(source_path: &PathBuf, target_path: &PathBuf) -> io::Result<()> {
     match read_dir(source_path) {
         Ok(entry_iter) => {
-            try!(create_dir_all(target_path));
+            create_dir_all(target_path)?;
             for entry in entry_iter {
-                let entry = try!(entry);
+                let entry = entry?;
                 let source_path = entry.path();
```

Comment:
At first this seemed like a no brainer, but if I try:

```rust
            for entry in entry_iter {
                let source_path = entry?.path();
                let target_path = target_path.join(entry?.file_name());
                add_resources(&source_path, &target_path)?;
            }
```

```bash
error[E0382]: use of moved value: `entry`
  --> build.rs:24:52
   |
23 |                 let source_path = entry?.path();
   |                                   ----- value moved here
24 |                 let target_path = target_path.join(entry?.file_name());
   |                                                    ^^^^^ value used here after move
   |
   = note: move occurs because `entry` has type `std::result::Result<std::fs::DirEntry, std::io::Error>`, which does not implement the `Copy` trait
```

Looks like you can't `?` a variable binding twice. If you see a way to handle this more cleanly, please let me know and I'm happy to make the change.